api/tests/test_models.py

```diff
@@ -1,40 +1,32 @@
 import pytest
-from django.utils import timezone
 
+from ..factories import PastEventFactory, FutureEventFactory, TodayEventsFactory
 from ..models import Event
 
 
 @pytest.mark.django_db
 def test_future_events():
     # given
-    today = timezone.now()
-
-    past_event = Event(name='name1', description='description1', datetime=today - timezone.timedelta(days=1))
+    past_event = PastEventFactory.create()
```

Comment:
FYI: this call is the same as `past_event = PastEventFactory()`. The latter is used more often in the codebase.

Reply:
In my case tests do not pass without use of 'create()' function.